testsuite/openmodelica/fmi/ModelExchange/2.0/fmi_attributes_06.mos

```diff
@@ -70,68 +70,12 @@ readFile("fmi_attributes_06.xml"); getErrorString();
 //   </ScalarVariable>
 //   <!-- Index of variable = \"5\" -->
 //   <ScalarVariable
-//     name=\"der(x)\"
-//     valueReference=\"4\"
-//     >
-//     <Real/>
-//   </ScalarVariable>
-//   <!-- Index of variable = \"6\" -->
-//   <ScalarVariable
-//     name=\"der(xd)\"
-//     valueReference=\"5\"
-//     >
-//     <Real/>
-//   </ScalarVariable>
-//   <!-- Index of variable = \"7\" -->
-//   <ScalarVariable
-//     name=\"der(y)\"
-//     valueReference=\"6\"
-//     >
-//     <Real/>
-//   </ScalarVariable>
-//   <!-- Index of variable = \"8\" -->
-//   <ScalarVariable
-//     name=\"der(yd)\"
-//     valueReference=\"7\"
-//     >
-//     <Real/>
-//   </ScalarVariable>
-//   <!-- Index of variable = \"9\" -->
-//   <ScalarVariable
 //     name=\"Fo\"
 //     valueReference=\"8\"
 //     >
 //     <Real/>
 //   </ScalarVariable>
-//   <!-- Index of variable = \"10\" -->
-//   <ScalarVariable
-//     name=\"x\"
```

Comment:
These states and corresponding derivatives shouldn't be suppressed.

Reply:
the variable x is a dummy state variable denoted by BackendDAE.DUMMY_STATE(), and its corresponding derivate der(x) and hence there is no derivative reference for the der(x), should dymmy states and derivative vars filtered or not ?